### PR TITLE
fix python version requirement

### PR DESCRIPTION
**Pull request by timcharper**
The docs say we need 3.4, but then make tells me 3.5.

One of these is wrong.

```
 ~/src/m8e/dcos-cli [master|…1]
14:59 $ make env
bin/env.sh
Cannot find supported python version 3.5. Exiting...
make: *** [env] Error 1
```

### CODE REVIEW

**Comment by timcharper** [comment]
Also added commit to make an error message easier to act on

File: dcos/http.py [hunk 1]
```diff
@@ -97,7 +97,7 @@ def _request(method,
     except requests.exceptions.SSLError as e:
         logger.exception("HTTP SSL Error")
         msg = ("An SSL error occurred. To configure your SSL settings, "
-               "please run: `dcos config set core.ssl_verify <value>`")
```
Comment (tamarrow) [inline dcos/http.py]: This is not exactly correct. ssl_verify can be True|False or path to CA_BUNDLE. It is the same as `verify` here: http://docs.python-requests.org/en/master/user/advanced/?highlight=ssl#ssl-cert-verification

**Comment by tamarrow** [comment]
@timcharper thanks for the pr! One small comment and then you'll need to update this tests here: https://github.com/dcos/dcos-cli/blob/master/cli/tests/integrations/test_ssl.py#L113

File: dcos/http.py [hunk 2]
```diff
@@ -100,7 +100,8 @@ def _request(method,
                "please run: `dcos config set core.ssl_verify <value>`")
         description = config.get_property_description("core", "ssl_verify")
         if description is not None:
-            msg += "\n<value>: {}".format(description)
+            msg += "\n\ncore.ssl_verify: {}".format(description)
```
Comment (tamarrow) [inline dcos/http.py]: Lets keep this line the same. We print out `"please run: `dcos config set core.ssl_verify <value>`` right before this, so it makes sense to explain what "<value>" is.

File: dcos/http.py [hunk 3]
```diff
@@ -100,7 +100,8 @@ def _request(method,
...
+        msg += "\n\npossible values: `True`, `False`, or a Path to CA"
```
Comment (tamarrow) [inline dcos/http.py]: Lets lowercase 'Path' and remove one of the newlines. 
CA should be `path to a CA_BUNDLE file`

**Comment by tamarrow** [comment]
@timcharper some small nits and don't forget to update `test_ssl.test_verify_ssl_without_cert_env_var`.

**Comment by timcharper** [comment]
You got it!

I couldn't figure out how to run the test_ssl tests properly; looks like I need to set up a snakeoil domain and server or something. Anyways, I'll check back in a bit to see if the build still fails and if I need to do anything else.

File: cli/tests/data/dcos.toml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [core]
+timeout = 5
 dcos_url = "http://dcos.snakeoil.mesosphere.com"
```
Comment (tamarrow) [inline cli/tests/data/dcos.toml]: please remove changes to this file

File: dcos/http.py [hunk 5]
```diff
@@ -97,10 +97,11 @@ def _request(method,
     except requests.exceptions.SSLError as e:
         logger.exception("HTTP SSL Error")
         msg = ("An SSL error occurred. To configure your SSL settings, "
-               "please run: `dcos config set core.ssl_verify <value>`")
+               "please run: `dcos config set core.ssl_verify <value>`, where "
+               "<value> is `True`, `False`, or a path to a CA_BUNDLE file.")
         description = config.get_property_description("core", "ssl_verify")
```
Comment (tamarrow) [inline dcos/http.py]: @timcharper if we change the description here: https://github.com/dcos/dcos-cli/blob/3c95b2b80774d9d7b2888d7c88f4cdd42a405fe9/dcos/data/config-schema/core.json#L39 than we don't have to change this file at all :)

**Comment by tamarrow** [comment]
@timcharper anything I can help with?

File: cli/tests/data/dcos.toml (hunk 4 above)
Comment (timcharper) [inline cli/tests/data/dcos.toml]: Done

File: dcos/http.py (hunk 1 above)
Comment (timcharper) [inline dcos/http.py]: Done

File: dcos/http.py (hunk 5 above)
Comment (timcharper) [inline dcos/http.py]: Done

**Pull request merged**